### PR TITLE
Paypal mint updates

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> includes updates to referral and reward table in order to support the new package, `@saasquatch/paypal-components`

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: https://saasquatch.atlassian.net/browse/DEV-3344
- Process.st launch checklist: https://app.process.st/runs/PayPal-Mint-Component-updates-lNnGSXnyAuIGOZa9C-1EMQ

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist

### Code review

- [x] Changes have been reviewed by at least one other engineer
- [x] Security impacts of this change have been considered

### CODE REVIEW

File: packages/mint-components/CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.6.5] - 2022-11-23
```
Comment (DerekSiemens) [inline packages/mint-components/CHANGELOG.md:10]: Noting date is going to need to be updated tomorrow

File: packages/mint-components/CHANGELOG.md [hunk 2]
```diff
@@ -28,7 +40,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - Replaced `<sqm-portal-container>` used in the view in order to change the flex direction of the name inputs when in mobile view.
     - Added missing margin bottom to the success alert
   - \<sqm-stat-container>
-    - Added media query to hide the stats right border when the screen size is 430px or less. 
+    - Added media query to hide the stats right border when the screen size is 430px or less.
   - \<sqm-hero>
     - Adding two column layout example for usage in the microsite/widget editor
```
Comment (DerekSiemens) [inline packages/mint-components/CHANGELOG.md:46]: Missing tagged version links at the bottom of the file

**Comment by DerekSiemens** [comment]
@00salmon PR looking good, please fill in the PR body though

File: packages/mint-components/CHANGELOG.md [hunk 3]
```diff
@@ -7,11 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.6.5] - 2022-11-24
```
Comment (DerekSiemens) [inline packages/mint-components/CHANGELOG.md:10]: Flagging date change needed

**Comment by DerekSiemens** [comment]
Looks good from a hygiene perspective but lets make sure that you get some peer review as well

File: packages/mint-components/src/components/sqm-rewards-table/sqm-rewards-table.tsx [hunk 4]
```diff
@@ -206,6 +207,7 @@ function useRewardsTableDemo(
     const rows =
       cellsPromise && (await Promise.all(cellsPromise)).filter((i) => i);
 
+    // @ts-ignore
```
Comment (Locrian24) [inline packages/mint-components/src/components/sqm-rewards-table/sqm-rewards-table.tsx:210]: Flagging ignore

**Review by Locrian24 — APPROVED** [review]
Just a ts-ignore to consider. Non-blocking tho

File: packages/mint-components/src/components/sqm-rewards-table/sqm-rewards-table.tsx (hunk 4 above)
Comment (00salmon) [reply packages/mint-components/src/components/sqm-rewards-table/sqm-rewards-table.tsx:210]: 👍 i think a typescript updated has caused this to fail when it didn't previously, not going to worry about it at the moment 😢

**Pull request merged**